### DISCUSSION TITLE
Profile and image layout discussion

### DISCUSSION THREAD

**Discussion by Thaumaturgy** [restincode/restincode#175] [Ideas]
Would like to see a stronger layout for the photos/images - possibly moving them to the top of the page. Make them a banner/ ribbon with scroll arrows so that there are 4 photos max visible at a time. Alternatively, Create a profile picture for each person in the list and place that at the top (right or left) and then leave the images at the bottom (resized/scaled to a common size but can be selected and viewed at original size)

![profile image](https://user-images.githubusercontent.com/53869752/65047164-f9413000-d92f-11e9-853f-01e985fec192.png)

**Comment by philcryer** [comment]
Could I suggest we look at a static site content option, say Jekyll or Hugo for a reorg, or is that a separate ticket/effort?

**Comment by mauvehed** [comment]
PR's are always open for anyone who has the time to put in effort to make improvements!

**Comment by mubix** [comment]
@philcryer @Thaumaturgy do either of you have the time/skill set to make it happen?

**Comment by philcryer** [comment]
Rob, I certainly have the interest, I think I would want to reformat things so it would be easier to maintain going forward, namely reworking all the data in Markdown, so you could run it through something like Hugo. This way you'd have a static site built quickly,in a way that a lot of people could understand. I don't know if this approach would be acceptable to people attached to this project, but if interested I could branch this and sketch something out; taking a few of these people and having a site built in that format.

It's one of those things that would cost a lot up front to migrate all the data, but would deal with the perceived (at least by me) complexity of how this is currently setup. The upside is that it would also address @Thaumaturgy comments on changing the layout, while keeping things open to reformatting with different themes/layouts in the future.

Phil

**Reply by mauvehed** [reply]
You need not rebuild the site 100% to what it is right now with regards to user data. I myself am not a front-end person, but I have used hugo/jekyll on plenty of other sites and its simple enough to adapt. 

What we really need is community members to jump in where they can, such as stubbing out the site itself, and others can assist with the data migration for the deceased. I would probably suggest starting a pull request for this that we can work on over time.
